taxonomies/categories.txt

```diff
@@ -31218,6 +31280,13 @@ en:Coconut butters, coconut purees, Coconut butter, coconut puree,
 fr:Purées de noix de coco, Purée de noix de coco, Beurres de noix de coco, Beurre de noix de coco, Purées de coco, Purée de coco, Beurres de coco, Beurre de coco
 nl:Kokosboter' kokosboters
 
+<en:Vegetable fats
+en:Cocoa butter
+fr:Beurre de cacao
+ciqual_food_code:en:16030
+ciqual_food_name:en:Cocoa butter
+ciqual_food_name:fr:Beurre de cacao
+
```

Comment:
<en:Vegetable fats
en:Cocoa butter
de:Kakaobutter
es:Mantecas de cacao
fr:Beurres de cacao
it:Burro di cacao
nl:Cacaoboters
wikidata:en:Q251106

Reply:
Merci, corrigé sur le prochain commit